StitchApp/RegressionTest/RegressionTestAppFunctions/insertJobinProdDB/source.js

```diff
@@ -0,0 +1,41 @@
+exports = function(payloadArg, jobTitle, jobUserName, jobUserEmail){
```

Comment:
We don't want to be writing to the prod and dev databases from the regression tests app for a number of reasons:

1. It doesn't test our job creation infrastructure -- which is an important part of our build system and which does change from time to time/require regression testing.
2. It opens up a potential vulnerability

Can we move the execution of the job initialization to be triggered by a webhook that is dedicated to regression tests and runs from within the environment?